src/components/organisms/DialogReply/DialogReply.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
```

Comment:
This component is missing tests, but our convention is that all components should be tested.